dev/integration_tests/deferred_components_test/run_release_test.sh

```diff
@@ -32,14 +32,15 @@ java -jar $bundletool_jar_path install-apks --apks=build/app/outputs/bundle/rele
 
 $adb_path shell "
 am start -n io.flutter.integration.deferred_components_test/.MainActivity
-sleep 20
+sleep 30
```

Comment:
🥲

Reply:
I am not certain this is useful, but it is not significant overhead to try. I will likely bring the number back down if it proves to be non-impactful.

Reply:
fwiw, this is generally against our style guide. Instead we prefer to use a signal to indicate that we are ready, e.g. having the test send a message when it is loaded.

Reply:
makes sense. @GaryQian mentioned there was an issue with the message signal. Was it difficult to do in bash?